projects/plugins/jetpack/modules/shortcodes/youtube.php

```diff
@@ -369,6 +373,22 @@ function youtube_id( $url ) {
 
 	}
 
+	/**
+	 * Format output for Calypso Reader/Notifications/Comments
+	 */
+	if ( defined( 'IS_WPCOM' ) && IS_WPCOM ) {
+		require_once WP_CONTENT_DIR . '/lib/display-context.php';
+		$context = Display_Context\get_current_context();
+		if ( Display_Context\NOTIFICATIONS === $context ) {
```

Comment:
Similar as in the comment above.

```suggestion
		if ( A8C\Display_Context\NOTIFICATIONS === $context ) {
```